microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -295,124 +373,120 @@ See also [DevTools: Focus Mode UI](https://github.com/MicrosoftEdge/MSEdgeExplai
+
+<!-- ???more needed, probably with an image, Detached Elements tutorial is in progress??? -->
+
 
 <!-- ====================================================================== -->
-<!-- ## VS Code themes for the DevTools -->
+## VS Code themes for the DevTools
 <!-- present in 96, 98 -->
 
+Enable this experiment to use Visual Studio themes in DevTools. For more information, see [Apply color themes to DevTools](../customize/theme.md).
+
 
 <!-- ====================================================================== -->
 ## Open source files in Visual Studio Code
 <!-- present in 96, 98 -->
 
-The **Open source files in Visual Studio Code** experiment replaces the code editor of the Sources tool with Visual Studio Code, for editing local files.  When you turn on this experiment, Developer Tools detects when you edit a local file, and prompts you to select a folder to use as your Workspace.
+The **Open source files in Visual Studio Code** experiment replaces the code editor of the Sources tool with Visual Studio Code, for editing local files. When you turn on this experiment, Developer Tools detects when you edit a local file, and prompts you to select a folder to use as your Workspace.
 
-After you select a folder to use as your Workspace, selecting any link to a file in DevTools opens the file in Visual Studio Code, rather than in the code editor of the Sources tool in DevTools.
+When you select a folder to use as your Workspace, selecting any link to a file in DevTools opens the file in Visual Studio Code. This action previously opened the file in the code editor of the Sources tool in DevTools.
 
 :::image type="content" source="../media/experiment-sources-in-code-editor-open.msft.png" alt-text="Selecting a file link in the Styles tool opens the file in Visual Studio Code" lightbox="../media/experiment-sources-in-code-editor-open.msft.png":::
 
-Any edits that you make in DevTools will now change the file on the hard drive and sync live with Visual Studio Code.  You can read about setting up your workspace in [Opening source files in Visual Studio Code](../sources/opening-sources-in-vscode.md).
+Any edits that you make in DevTools now change the file on the hard drive and sync live with Visual Studio Code. You can read about setting up your workspace in [Opening source files in Visual Studio Code](../sources/opening-sources-in-vscode.md).
 
 This experiment is available starting with Microsoft Edge version 96.
 
 
 <!-- ====================================================================== -->
-<!-- Microsoft Edge automatically pretty print in the Sources Panel -->
-<!-- present in 96, 98 -->
-
-
-<!-- ====================================================================== -->
-<!-- ====================================================================== -->
-<!-- "WARNING: These experiments are particularly unstable:" -->
+>> [!WARNING]
```

Comment:
We will need to remove this I think. There's no content displayed after it, so the warning is not useful anymore.

Reply:
agreed